docs/tools/config_system.md

```diff
@@ -181,20 +251,17 @@ The last point above is important. The application can freely override the confi
     },
     "target_overrides": {
         "*": {
-            "target.serial_console_speed": 2400,
+            "mylib.queue_size": 2400,
             "mylib.timer_period": 100
         },
-        "Base": {
-            "target.serial_console_speed": 9600
+        "K64F": {
+            "mylib.queue_size": 9600
         }
     }
 }
 ```
 
-`target_overrides` works a lot like it does in libraries, but there are a few differences:
-
-- Because the application can override any configuration parameter, it must specify configuratoin parameters using their prefix (like `mylib.timer_period`). If an overridden parameter doesn't have a prefix, the config system assumes that the overridden paramater is one of the parameters the application defines in its own `config` section.
-- The `*` key in `target_overrides` will match *any* target. It is possible to use the `*` key in a library's `target_overrides` too, but it'd make little sense to do so, since it will always override the values defined in the library's `config` section. In an application it might make sense to use the `*` key, since it can be used to override the configuration defined by the target or the dependent libraries, no matter which target is used for building.
+The application may override any configuration parameter by specifying the configuration parameters including their prefix (like `mylib.timer_period`). If an overridden parameter doesn't have a prefix, it overrides a parameter in its own `config` section.
```

Comment:
This section needs to be updated. There is no `target.serial_console_speed` introduced in the example. Also what is the `Base` target that is being referred to?